### ISSUE TITLE
NTR: Muscle biopsy: lactate dehydrogenase low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9081]
New term request Muscle biopsy: lactate dehydrogenase low (segal_050203184833): LDH

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000196
- label: Reduced muscle lactate dehydrogenase activity
- parents: [Abnormal enzyme activity in muscle tissue (HP:0034526)](https://hpo.jax.org/app/browse/term/HP:0034526)
- pmid: 
- def: Activity of the enzyme lactate dehdrogenase (LHD; 1.1.1.27) in muscle tissue is below the lower limit of normal. LDH catalyzes the conversion of pyruvate to lactate and back, as it converts NAD+ to NADH and back.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9081
- synonyms: Muscle biopsy: lactate dehydrogenase low(NONE)
